src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.Git.cs

```diff
@@ -190,21 +193,51 @@ private static Branch SwitchToBranchAndUpdate(string branchToSwitchTo, string ba
             return Enlistment.Head;
         }
 
-        private static void CreateDummyCommit(CancellationToken cancellationToken)
+        private static async Task<GitPullRequest> CreatePlaceholderBranchAsync(CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
-            var message = $"DUMMY INSERTION FOR {Options.InsertionName}";
-            var options = new CommitOptions()
+            var gitClient = ProjectCollection.GetClient<GitHttpClient>();
+            var repository = await gitClient.GetRepositoryAsync(project: Options.TFSProjectName, repositoryId: "VS", cancellationToken: cancellationToken);
+
+            var refs = await gitClient.GetRefsAsync(
+                repository.Id,
+                filter: $"heads/{Options.VisualStudioBranchName}",
+                cancellationToken: cancellationToken);
+            GitRef sourceBranch = refs.Single();
+
+            var branchName = GetNewBranchName();
+
+            _ = await gitClient.CreatePushAsync(new GitPush()
             {
-                AllowEmptyCommit = true
-            };
-            var watch = Stopwatch.StartNew();
-            var commit = Enlistment.Commit(
-                message,
-                InsertionToolSignature,
-                InsertionToolSignature,
-                options);
-            Console.WriteLine($"Committing took {watch.Elapsed.TotalSeconds} seconds");
+                RefUpdates = new[] {
+                    new GitRefUpdate()
+                    {
+                        Name = $"refs/heads/{branchName}",
+                        OldObjectId = sourceBranch.ObjectId,
+                    }
+                },
+                Commits = new[] {
+                    new GitCommitRef()
+                    {
+                        Comment = $"PLACEHOLDER INSERTION FOR {Options.InsertionName}",
+                        Changes = new GitChange[]
+                        {
+                            new GitChange()
+                            {
+                                ChangeType = VersionControlChangeType.Delete,
+                                Item = new GitItem() { Path = "/Init.ps1" }
```

Comment:
what happens if we ever introduce a file called /Init.ps1?

Is git ok with creating a commit that deletes a non existent file?

Reply:
We have a file called `Init.ps1` -- it's the main file! I include this change so that the tests fail and the PR doesn't accidentally get merged. If we ever end up changing the name of the entry script, this might fail, but that seems like a rare/unlikely change.